client/src/components/ProfileSettings/Availability/useStyles.ts

```diff
@@ -0,0 +1,5 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({}));
```

Comment:
If we not using this file, please delete it?

Reply:
I've added TODO comments to useStyles files instead.